src/group/groupService.ts

```diff
@@ -132,14 +119,18 @@ export const fetchGroups = () => {
       ])
       .then(groups =>
         groups.map(edge => ({
-          ..._.omit(['accountMembership', 'membershipsCount'], edge.node),
+          ..._.omit(
+            ['memberships', 'accountMembership', 'membershipsCount'],
+            edge.node
+          ),
           membersInfo: extractMembersInfo(edge.node),
         }))
       )
       // eslint-disable-next-line lodash-fp/no-extraneous-function-wrapping
-      .then(groups =>
-        _.orderBy([group => group.name.toLowerCase()], [], groups)
-      )
+      .then(groups => {
+        dispatch(setMemberships(getMemberships(groups)));
```

Comment:
it's because i've separated the Terraso Group & membership logic into different services & slices since we aren't moving the Terraso Group logic into the shared code. before, the group slice would be able to update the group info and membership info within the same slice in the same action resolver. but slices must communicate with each other by dispatch. this is already how the landscape slice was working, it just updates group slice to match. 

while checking on the answer to this question i did notice that the landscape slice does this extra dispatch logic in the slice code instead of the service code, so i may update the group code to match when i rebase for consistency (i think it does make sense to keep dispatch code out of the service code).

you're right the eslint comment doesn't make sense. we are using eslint afaik, i think it runs on each PR to the repo